mixer/pkg/attribute/envoyProtoBag.go

```diff
@@ -168,6 +168,13 @@ func AccessLogProtoBag(msg *accesslog.StreamAccessLogsMessage, num int) *EnvoyPr
 
 	reqMap["context.proxy_error_code"] = ParseEnvoyResponseFlags(commonproperties.GetResponseFlags())
 
+	//This is for identifying the log type as Service Access Logs instead of Mixer Report
```

Comment:
nit: can you add a space between '//' and first character in the comment? The same to all the following lines.